core/utils.py

```diff
@@ -20,5 +20,29 @@ def wrapper(self, request, *args, **kwargs):
             return JsonResponse({'ERROR' : 'INVALID_USER'}, status=400)
             
         return func(self, request, *args, **kwargs)
+    return wrapper
+
+def SigninCheckDecorator(func):
+    def wrapper(self, request, *args, **kwargs):
+        try:
+            token        = request.headers.get('Authorization', None)
+
+            if not token:
+                request.user = 0
+                return func(self, request, *args, **kwargs)
+                
+            payload      = jwt.decode(token, SECRET_KEY, algorithms=ALGORITHM)
+            user         = User.objects.get(id = payload['user_id'])
+            request.user = user
```

Comment:
굳이 여러번에 걸쳐서 변수 지정할 필요는 없어 보입니다.
```suggestion
            request.user = User.objects.get(id = payload['user_id'])
```

Reply:
수정완료하였습니다!